assignments/callbacks.js

```diff
@@ -24,23 +24,33 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  cb(arr.length);
```

Comment:
you would actually jut want to pass the array itself into the cb, not arr.length